.github/workflows/docker.yml

```diff
@@ -0,0 +1,53 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    tags: ["*"]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
```

Comment:
I guess here we would need to adapt the docker name?

Reply:
I guess it would be: `av_imu`

Reply:
Renamed github repo so this is variable value should now be as expected: `av_imu`